OracleInstantClient/README.md

```diff
@@ -1,41 +1,40 @@
 # About this Docker Image
 
-This Docker image contains the Oracle Instant Client 'Basic', 'SDK' and 'SQL*Plus' packages.  It can be extended to run OCI, OCCI, and JDBC applications.  It can also be extended to build and run scripting language drivers that use OCI such as Python's cx_Oracle, Node.js's node-oracledb, PHP's OCI8, and Ruby's ruby-oci8.  
+These Docker images contain the Oracle Instant Client 'Basic', 'SDK' and 'SQL*Plus' packages.  They can be extended to run OCI, OCCI, and JDBC applications.  They can also be extended to build and run scripting language drivers that use OCI such as Python's cx_Oracle, Node.js's node-oracledb, PHP's OCI8, and Ruby's ruby-oci8.  
```

Comment:
What about using "Oracle Call Interface (OCI)" and "Oracle C++ Call Interface (OCCI)" to help differentiate from Cloud OCI?